### PR TITLE
Better support for stop sequences

### PR DESCRIPTION
**Pull request by xl0**
Show the stop reason in the summary
Add options stop= to client that accepts str or list[str]
Add use examples

### CODE REVIEW

File: settings.ini [hunk 1]
```diff
@@ -6,7 +6,7 @@ min_python = 3.8
 license = apache2
 black_formatting = False
 requirements = fastcore>=1.5.33 anthropic toolslm
-dev_requirements = google-auth
+dev_requirements = google-auth boto3
```
Comment (jph00) [inline settings.ini:9]: What's boto3 added for?

**Review by jph00 — CHANGES_REQUESTED** [review]
This is great! Just one question.

File: settings.ini (hunk 1 above)
Comment (xl0) [reply settings.ini:9]: The AWS examples at the bottom need it.

Comment (jph00) [reply settings.ini:9]: Could we replace those with something that doesn't bring in an additional dev dep please?

Comment (xl0) [reply settings.ini:9]: Without boto3 Claude can't use AWS:
image

https://github.com/anthropics/anthropic-sdk-python/blob/main/src/anthropic/lib/bedrock/_auth.py#L21

They don't include it as a dependency, but if you want the AWS example to work, we need boto3.

Comment (xl0) [reply settings.ini:9]: I'm ok with just ignoring this and not including the dependency either.

Comment (jph00) [reply settings.ini:9]: Thank you -- sorry I was a bit slow, I understand the issue now!

Comment (jph00) [inline settings.ini:9]:
```suggestion
dev_requirements = google-auth
```

**Pull request merged**